scripts/globals/job_utils/rune_fencer.lua

```diff
@@ -0,0 +1,54 @@
+-----------------------------------
+-- Rune Fencer Job Utilities
+-----------------------------------
+require("scripts/settings/main")
+require("scripts/globals/ability")
+require("scripts/globals/status")
+require("scripts/globals/msg")
+require("scripts/globals/weaponskills")
+require("scripts/globals/jobpoints")
+-----------------------------------
+xi = xi or {}
+xi.job_utils = xi.job_utils or {}
+xi.job_utils.rune_fencer = xi.job_utils.rune_fencer or {}
+-----------------------------------
+
+local function getRUNLevel(player)
+    return player:getMainJob() == xi.job.RUN and player:getMainLvl() or player:getSubLvl()
```

Comment:
This function will return SubLvl regardless if it is RUN or not if RUN is not set to the main job.  Might need to rework the logic on this.

Reply:
Short circuit logic isn't very readable for the layperson here anyway, better of expanding it out
```lua
if player:getMainJob() == xi.job.RUN then
    return player:getMainLvl()
else
    -- must be sj then
    return player:getSubLvl()
end
```